drivers/python/rethinkdb/ast.py

```diff
@@ -1483,7 +1486,7 @@ class Sync(RqlMethodQuery):
     st = 'sync'
 
 
-class Branch(RqlTopLevelQuery):
+class Branch(RqlMethodQuery):
```

Comment:
I think this only determines how the term is pretty printed. I'd prefer keeping the old behavior here.